### PR TITLE
Fix mget call with empty subscribers

### LINKED ISSUE #1757: ERR wrong number of arguments for 'mget' command
Laravel 8
Php 8
Postgresql 13

I am trying to implement subscription as specified in docs but after setup, when i try to run mutation

```

<?php

namespace App\GraphQL\Mutations;

use App\Models\GroupMessage;
use Nuwave\Lighthouse\Execution\Utils\Subscription;

class SendGroupMessage
{
    /**
     * @param  null  $_
     * @param  array<string, mixed>  $args
     */
    public function __invoke($_, array $args)
    {
        extract($args);

        $message =  GroupMessage::create([
            'sender_id' => auth()->id(),
            'poll_id' => $poll_id,
            'message' => $message
        ]);

        Subscription::broadcast('groupMessageCreated', $message);

        return $message;
    }
}
```

I am getting this error:-

### PR DESCRIPTION
**Pull request by thekonz**
- [x] Added or updated tests
- [x] Documented user facing changes
- [x] Updated CHANGELOG.md

Resolves #1757 

**Changes**

Fix the mget call when there are no subscribers

### CODE REVIEW

**Comment by spawnia** [comment]
@thekonz thanks for digging in and owning up to the code you contributed :) Invited you as a collaborator, you can join our Slack so I can invite you to our collaborators channel, too https://join.slack.com/t/lighthouse-php/shared_invite/zt-4sm280w1-wu21r94f3kLRRtBXRbXVfw

**Comment by thekonz** [comment]
Not sure what is happening with the schema tests, but I could reproduce the mget issue with a test and the early return in the storage manager fixes it.

**Comment by spawnia** [comment]
Btw, fixing the test failure in https://github.com/nuwave/lighthouse/pull/1760

File: src/Subscriptions/Storage/RedisStorageManager.php [hunk 1]
```diff
@@ -64,6 +64,9 @@ public function subscribersByTopic(string $topic): Collection
         // Since we store the individual subscribers with a prefix,
         // but not in the set, we have to add the prefix here.
         $subscriberIds = array_map([$this, 'channelKey'], $subscriberIds);
+        if (empty($subscriberIds)) {
```
Comment (spawnia) [inline src/Subscriptions/Storage/RedisStorageManager.php:67]:
```suggestion
        if (count($subscriberIds) === 0) {
```

File: src/Subscriptions/Storage/RedisStorageManager.php [hunk 2]
```diff
@@ -64,6 +64,9 @@ public function subscribersByTopic(string $topic): Collection
         // Since we store the individual subscribers with a prefix,
         // but not in the set, we have to add the prefix here.
         $subscriberIds = array_map([$this, 'channelKey'], $subscriberIds);
+        if (count($subscriberIds) === 0) {
```
Comment (spawnia) [inline src/Subscriptions/Storage/RedisStorageManager.php:67]: Can we do this return even earlier?

File: src/Subscriptions/Storage/RedisStorageManager.php (hunk 1 above)
Comment (thekonz) [reply src/Subscriptions/Storage/RedisStorageManager.php]: Any reason for that?

File: src/Subscriptions/Storage/RedisStorageManager.php (hunk 2 above)
Comment (thekonz) [reply src/Subscriptions/Storage/RedisStorageManager.php:67]: Oh yes we could, true.

File: src/Subscriptions/Storage/RedisStorageManager.php (hunk 1 above)
Comment (spawnia) [reply src/Subscriptions/Storage/RedisStorageManager.php]: Yup, empty is just generally too loose.

**Review by spawnia — APPROVED** [review]
Nice work, thank you

**Pull request merged**